relay-event-schema/src/protocol/span_v2.rs

```diff
@@ -0,0 +1,388 @@
+use opentelemetry_proto::tonic::trace::v1::{
+    span::SpanKind as OtelSpanKind, status::StatusCode as OtelSpanStatus,
+};
+use relay_protocol::{Annotated, Empty, FromValue, IntoValue, Object, Value};
+
+use std::fmt;
+use std::str::FromStr;
+
+use serde::Serialize;
+
+use crate::processor::ProcessValue;
+use crate::protocol::{Attribute, SpanId, Timestamp, TraceId};
+
+use super::OperationType;
+
+/// A version 2 (transactionless) span.
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue)]
+pub struct SpanV2 {
+    #[metastructure(required = true, trim = false)]
+    pub trace_id: Annotated<TraceId>,
+
+    /// The ID of the span enclosing this span.
+    pub parent_span_id: Annotated<SpanId>,
+
+    /// The ID of the trace the span belongs to.
+    /// The Span id.
+    #[metastructure(required = false, trim = false)]
+    pub span_id: Annotated<SpanId>,
+
+    /// Span type (see `OperationType` docs).
+    #[metastructure(max_chars = 128)]
```

Comment:
Any reason for the limit?

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/v1.44.0/specification/trace/api.md#span

Reply:
Removed it for now, we can revisit later.